semantic_conventions/resource/container.yaml

```diff
@@ -32,3 +32,31 @@ groups:
         brief: >
           Container image tag.
         examples: ['0.1']
+      - id: image.id
+        type: string
+        brief: >
+          Runtime specific image identifier. Usually a hash algorithm followed by a UUID.
+        note:
+          On docker, `image.id` corresponds to the `Image` field from the Docker container inspect [API](https://docs.docker.com/engine/api/v1.43/#tag/Container/operation/ContainerInspect) endpoint.
```

Comment:
nit:
```suggestion
          On docker, `conainer.image.id` corresponds to the `Image` field from the Docker container inspect [API](https://docs.docker.com/engine/api/v1.43/#tag/Container/operation/ContainerInspect) endpoint.
```

Reply:
Done in 1845233fde0d09c2858ed1bbfc20cde92ca1530d.